### PR TITLE
Setup ClerkJS

### PR DESCRIPTION
**Pull request by Zack-Ren**
Refactored Navbar into it's own component to allow integration with Clerk <UserButton/>

### CODE REVIEW

File: web/components/navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          <NavbarContent justify="end">
+            <NavbarItem>Help</NavbarItem>
+            <NavbarItem>
+              <UserButton />
+            </NavbarItem>
+          </NavbarContent>
+        </Navbar>
+      </SignedIn>
+      <SignedOut>
+        <RedirectToSignIn />
```
Comment (Lev1ty) [inline web/components/navbar.tsx:29]: would this make more sense to be in layout.tsx?

Comment (Zack-Ren) [reply web/components/navbar.tsx:29]: It has to be in a client component

Comment (Lev1ty) [reply web/components/navbar.tsx:29]: what about putting in the providers component?

Comment (Lev1ty) [reply web/components/navbar.tsx:29]: actually, that would require all routes to be authenticated...
makes sense leaving as-is

**Review by Lev1ty — APPROVED** [review]

**Review by sunb26 — APPROVED** [review]

**Pull request merged**